doc/INSTALL-Win64.txt

```diff
@@ -0,0 +1,178 @@
+-------------------------------------------------------------------------------
+KVIrc4 win64 compiling howto                                          by Cizzle
+ based on wodim's guide at https://thacid.wordpress.com/2010/10/18/compiling-kvirc-64-bits-from-svn-using-visual-studio-2008/
+ and DarthGandalf's appveyor buildinstructions
+-------------------------------------------------------------------------------
+
+This paper describes how to successfully compile a working copy of KVIrc 4.3.x
+under Microsoft Windows operating systems. For own compilations we assume a
+root-directory C:\kvirccomp\ being used.
+
+-------------------------------------------------------------------------------
+Prerequisites
+-------------------------------------------------------------------------------
+
+- A 64-bit Microsoft Windows operating system.
+	Should work at least on 7
+
+- Microsoft Visual Studio 2013 (Community Edition).
+	Get it from https://www.visualstudio.com/nl-nl/products/visual-studio-community-vs
+	Install in default location with default options, though you can slim it down a lot if wanted.
+
+- CMake.
+	Get it from http://www.cmake.org/download/
+	Install in default location with default options and choose "Add CMake to command path for all users".
+
+- GIT client. 
+	Get one from https://git-for-windows.github.io/
+	Git bash should be enough.
+
+- Qt.
+	Get it from https://www.qt.io/download-open-source/
+	Version 5 should work.
+	Get the online installer and be sure to mark MSVS 2013 64bit version.
+	Install in default location with default options.
+
+- Perl.
+	Get it from http://www.activestate.com/activeperl/downloads
```

Comment:
Oh. I tried with strawberry perl and active perl, until found that .h magic which worked with strawberry perl.
So with active perl it's similar? interesting.